package.json

```diff
@@ -124,34 +120,45 @@
             {
                 "id": "hubl",
                 "aliases": ["HubL"],
-                "extensions": [
-                    ".hubl.html"
-                ],
+                "configuration": "./langconfig/hubl-language-configuration.json"
+            },
+            {
+                "id": "html-hubl",
+                "aliases": ["HTML + HubL"],
+                "configuration": "./langconfig/hubl-language-configuration.json"
+            },
+            {
+                "id": "css-hubl",
+                "aliases": ["CSS + HubL"],
                 "configuration": "./langconfig/hubl-language-configuration.json"
             }
         ],
         "grammars": [
             {
                 "language": "hubl",
                 "scopeName": "source.hubl",
-                "path": "./syntaxes/hubl.json"
+                "path": "./syntaxes/hubl.json",
+                "injectTo": [
+                    "text.html.hubl",
+                    "text.css.hubl"
+                ]
             },
```

Comment:
Do we need this grammar anymore? Can we just use `html-hubl` and `css-hubl` to prevent confusion when selecting the file type?

Reply:
As of right now the `hubl` grammar gets injected into `html-hubl` and `css-hubl` so it is still needed. But based on @gcorne's comment above, I want to dig into this to make sure that it's set up correctly.